packages/ui-portal/src/Portal/index.tsx

```diff
@@ -56,42 +80,20 @@ type State = {
 category: components/utilities
 ---
 @module Portal
+@tsProps
```

Comment:
This is the first notable change: if you want to turn on the Properties generation from the ts props, add this label in the description